src/constructorio-id.js

```diff
@@ -245,20 +246,29 @@
     this.session_id = sessionId;
     this.session_is_new = sessionData && sessionDataId === sessionId ? false : true;
 
+    // persist new status for when ciojs-client is instantiated before beacon
+    if (sessionData && sessionData.newToBeacon) {
+      this.new_to_beacon = true;
```

Comment:
We are setting this to true but we are not using it anywhere else. Am I missing something?

Reply:
This is utilized in a following [PR](https://github.com/Constructor-io/autocomplete-ui/pull/2573/files#diff-b2239cc4e4a25bd5d298e992b0949d07cbb19ab0205fcd0b9975b813203a20d3R176) in the autocomplete-ui